worker/dblogpruner/worker_test.go

```diff
@@ -54,8 +87,8 @@ func (s *suite) StartWorker(c *gc.C, maxLogAge time.Duration, maxCollectionMB in
 
 func (s *suite) TestPrunesOldLogs(c *gc.C) {
 	maxLogAge := 24 * time.Hour
-	noPruneMB := int(1e9)
-	s.StartWorker(c, maxLogAge, noPruneMB)
+	s.setupState(c, "24h", "1000P")
```

Comment:
It actually goes up to yottabytes. You never know how much status history you'll need ;)